docs/deployment.md

```diff
@@ -3,7 +3,18 @@
 ## MongoDB Operator Requirements
 
 To function properly, some requirements in the Kubernetes deployment must be met, and a MongoDB
-database instance must be set up.
+database instance dedicated to the Kubernetes cluster must be set up.
```

Comment:
Instead of "MongoDB database instance" I'd use the term "MongoDB cluster" to not get confused with [databases](https://www.mongodb.com/docs/compass/current/databases/) (=container of collections).

Reply:
Hmm, but it must not be a cluster. Could also be a single instance, e.g. in dev.

Reply:
This is the [wording by MongoDB](https://www.mongodb.com/basics/clusters):
> In contrast to a single-server MongoDB database, a MongoDB cluster allows a MongoDB database to either horizontally scale across many servers with sharding, or to replicate data ensuring high availability with MongoDB replica sets, therefore enhancing the overall performance and reliability of the MongoDB cluster.

Then I'm fine with the wording.